composer.lock

```diff
@@ -1170,6 +1170,90 @@
             "time": "2015-12-19 20:16:43"
         },
         {
+            "name": "league/flysystem",
```

Comment:
you actually messed up composer install
Your requirements could not be resolved to an installable set of packages.